README.md

```diff
@@ -94,7 +94,66 @@ Field Name | Type | Optional? | Description
 ```:request_id``` | String | Yes | Generated by the SDK when needed (*do not set*)
 ```:content_id``` | String | No | Identifier for the content to be shown, must be set.
 ```:properties``` | Properties | Yes | Any additional custom properties to associate. For v1 integrations, it is fine not to fill in all the properties.
+---
+### Size
+User's screen dimensions.
+Field Name | Type | Optional? | Description
+---------- | ---- | --------- | -----------
+```:width``` | Integer | No | Screen width
+```:height``` | Integer | No | Screen height
+---
+
+### ClientHints
```

Comment:
JFYI - I added client hints but learned that it's not very well supported.

Reply:
Yeah, https://deviceatlas.com/blog/postponed-googles-attempt-deprecate-and-freeze-user-agent-header :-)